src/actors/containerization/services/wordpress/wordpress_scanner/wordpress_scanner.py

```diff
@@ -0,0 +1,74 @@
+import json
+import os
+import re
+
+
+def get_version(root_dir):
+    pattern = '\d(.\d){1,2}'
+    version = ''
+
+    with open(os.path.join(root_dir, 'wp-includes/version.php')) as f:
+        for line in f:
```

Comment:
Instead of iterating over php files line by line, it's better to read and find ```$wp_version``` by regex

Reply:
By "better" I thought about having less lines (ofc without affecting code complexity), although for big files your solution is correct (however version.php is a small file). I was thinking about this again, so just leave it like that, I am fine with this.